common/elasticsearch/OWNERS

```diff
@@ -1,8 +0,0 @@
-reviewers:
```

Comment:
Is common/elasticsearch entirely for the purpose of elasticsearch sink? Are you removing the whole elasticsearch sink?

Reply:
Yes.